INFO     lingkuan_828.conftest:conftest.py:62 [2025-08-29 09:36:28] 用例 test_cloudTrader/test_cloudOrderClose.py::TestCloudClose::TestMasOrderSend10::test_copy_close_order2 开始登录，环境: uat
INFO     lingkuan_828.conftest:conftest.py:100 UAT登录尝试 1/5，MFA验证码: 781633
ERROR    lingkuan_828.conftest:conftest.py:129 第1次登录失败: 登录失败: MFA验证失败
INFO     lingkuan_828.conftest:conftest.py:100 UAT登录尝试 2/5，MFA验证码: 339253
INFO     lingkuan_828.conftest:conftest.py:124 UAT登录成功（第2次尝试）
INFO     lingkuan_828.commons.variable_manager:variable_manager.py:111 [2025-08-29 09:14] 运行时变量已保存到: D:\pycharm_test\lingkuan_828\VAR\runtime_vars_cloud.json
INFO     lingkuan_828.commons.api_base:api_base.py:152 [2025-08-29 09:36:56] POST请求（JSON）: /mascontrol/cloudTrader/cloudOrderClose | 数据: {'flag': 1, 'id': 118, 'cloudTraderId': [1583]}
INFO     lingkuan_828.commons.api_base:api_base.py:160 [2025-08-29 09:36:56] 请求后等待 1 秒
INFO     lingkuan_828.conftest:conftest.py:157 [2025-08-29 09:36:57] 用例 test_cloudTrader/test_cloudOrderClose.py::TestCloudClose::TestMasOrderSend10::test_copy_close_order2 执行完毕